setup/security/authorization/google-groups/index.md

```diff
@@ -52,22 +52,22 @@ create a service account that will access the G Suite Directory API.
 
 ## Configure with Halyard
 
-With the authorized service account's credentials in hand, use Halyard to configure Fiat:
-
-```bash
-ADMIN=admin@your.org              # An administrator's email address
-CREDENTIALS=/path/to/creds.json   # The downloaded service account credentials
-DOMAIN=your.org                   # Your organization's domain.
-
-hal config security authz google edit \
-    --admin-username $ADMIN \
-    --credential-path $CREDENTIALS \
-    --domain $DOMAIN
-
-hal config security authz edit --type google
-
-hal config security authz enable
-```
-
-
-## Troubleshooting
+1. Make sure you've configured roles for accounts, as described [here](/setup/security/authorization/#accounts). Each role included in the command is the name of a group in
+the organization.
```

Comment:
Each role included in the command should match the name of a group in the organization